File: web/src/layout/DrawableDemoXml.ts
//
// DrawableDemoXml.ts
// Created by Khrysalis XML Typescript
//
import { loadHtmlFromString, findViewById, getViewById, replaceViewWithId, startupAutoResize } from 'butterfly-web/dist/views/html'
import { customViewSetDelegate } from 'butterfly-web/dist/views/CustomView'
import htmlForDefault from './drawable_demo.html'
//! Declares com.lightningkite.butterflytemplate.layouts.DrawableDemoXml
export class DrawableDemoXml {
    xmlRoot!: HTMLElement
    loadHtmlString(): string {
        return htmlForDefault
    }
    setup(dependency: Window): HTMLElement {
        const view = loadHtmlFromString(this.loadHtmlString())
        this.xmlRoot = view
        return view
    }
}

File: android/src/main/java/com/lightningkite/butterflytemplate/layouts/DrawableDemoXml.kt
//
// DrawableDemoXml.kt
// Created by Khrysalis XML Android
//
package com.lightningkite.butterflytemplate.layouts

import android.widget.*
import android.view.*
import com.lightningkite.butterfly.views.widget.*
import com.lightningkite.butterfly.views.*
import com.lightningkite.butterfly.android.*
import com.lightningkite.butterflytemplate.R

class DrawableDemoXml {

    
    
    
    lateinit var xmlRoot: View

    fun setup(dependency: ActivityAccess): View {
        val view = LayoutInflater.from(dependency.context).inflate(R.layout.drawable_demo, null, false)
        return setup(view)
    }
    fun setup(view: View): View {
        xmlRoot = view
        
        
        
        
        return view
    }
}